docs/v7-migration.md

```diff
@@ -180,7 +180,7 @@ module.exports = {
 };
 ```
 
-### [Switch to `-proposal-` for TC39 Proposals](/blog/2017/12/27/nearing-the-7.0-release.html#renames-proposal)
+### [Switch to `-proposal-` for TC39 Proposals](https://babeljs.io/blog/2017/12/27/nearing-the-7.0-release.html#renames-proposal)
```

Comment:
The broken link detection is not quite working on docs <--> blog links. This is a known limitation: https://docusaurus.io/docs/markdown-features/links